components/Card.js

```diff
@@ -51,6 +51,32 @@ CardWithLoading.propTypes = {
     title: PropTypes.string,
 };
 
-CardWithLoading.defaultTypes = {
+CardWithLoading.defaultProps = {
     title: '',
 };
+
+export const CardWithError = ({ title, onClick }) => (
```

Comment:
Shouldn't we change the name of the props to something indicating its use ? Like `onFailureRetry`